app/models/user.rb

```diff
@@ -336,6 +336,10 @@ def ignores_notification?(notification)
     ignored_notifications.include? notification.subject
   end
 
+  def forum_messages
+    Message.where(sender: self).or(Message.where('sender = ?', uid)).where.not(discussion_id: nil)
+  end
```

Comment:
Awesome! I guess this is the line that will change in the next stage/version, right?

Reply:
We may turn into a plain has_manny in the future

Reply:
next stage is basically a revert of the last commit!

Reply:
> We may turn into a plain has_manny in the future

yeah, that's what it'll be after the revert :laughing: